### ISSUE TITLE
JSON parser crashes with trailing whitespace.

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#41]
The following JSON crashed the parser. The issue is due to trailing whitespace.

```
{
	"files": [
		"a.obs",
		"b.obs"
	],
	"libs": [
		"gen_collect.obl",
		"regex.obl",
		"net.obl",
		"misc.obl",
		"odbc.obl"
	],
	"flags": ""
}

```

**Comment by objeck** [comment]
fixed in v6.0.0

**Comment by objeck** [comment]
fixed in v6.0.0
